torch/_dynamo/guards.py

```diff
@@ -395,14 +414,14 @@ def strip_getattr_getitem(name):
 
 def get_verbose_code_part(code_part: str, guard: Guard) -> str:
     extra = ""
-    if guard.user_stack:
-        for fs in reversed(guard.user_stack):
-            if fs.filename not in uninteresting_files():
-                extra = f"  # {format_frame(fs, line=True)}"
-                break
-    elif guard.stack:
-        extra = f"  # {format_frame(guard.stack.summary()[-1])}"
-
+    if guard is not None:
```

Comment:
why can `guard` be None now?

Reply:
This is the same for `DuplicateInputs` guard (see below at line 2345). Not sure why that wasn't getting triggered. My guess is that there's no test for this?